wdl2cwl/main.py

```diff
@@ -344,6 +344,18 @@ def convert(workflow: str) -> str:
             )
         )
 
+    if "cpu" in ast.task_runtime:
+
+        cpu = ""
+        if '"' not in ast.task_runtime["cpu"]:
+            cpu = "$(inputs." + ast.task_runtime["cpu"] + ")"
```

Comment:
Okay, but what if `cpu` is just a plain number, no `"`?

Reply:
Right. I didn't think of that. I'll modify the code